stubs/docker-compose.yml

```diff
@@ -12,6 +12,8 @@ services:
             - '${APP_PORT:-80}:80'
         environment:
             WWWUSER: '${WWWUSER}'
+            XDEBUG: '${SAIL_XDEBUG:-false}'
+            XDEBUG_PORT: '${SAIL_XDEBUG_PORT:-9000}'
```

Comment:
I'd make the default 9003 here, that's xdebug3 default and is already setup in PhpStorm by default to listen on that port

Reply:
vscode extension defaults to 9000
i don't know about other IDE's

Reply:
PHPStorm Default to `9000,9003`